src/client/producer.ts

```diff
@@ -0,0 +1,203 @@
+import { ProducerRecord, RecordMetadata } from "kafkajs";
+import * as vscode from "vscode";
+import { addQueryParameter, ClientAccessor } from ".";
+import { Producer as KafkaJSProducer } from "kafkajs";
+
+
+export enum ProducerLaunchState {
+    none,

```

Comment:
how about `idle`? or `unknown`?

Reply:
fixed with `idle` and consumer state has been changed too.